README.md

```diff
@@ -20,53 +26,122 @@ to avoid command conflicts with other bots in the server, users can customize UH
 lines. The customizability ensures a seamless and non-disruptive experience across various 
 functionalities.
 
+## Installation Guide
+
+### Before You Start: Setting Up a Discord Server.
+
+In order to install UHelper, you have to create your own server or are already an administrator in 
+an existing server. If you have not used Discord before or do not know how to work with servers, it 
+is recommended that you read through [this beginner
+'s guide](https://support.discord.com/hc/en-us/articles/360045138571-Beginner-s-Guide-to-Discord#h_efc9b7bc-47bc-4212-8b9c-c0fa76573cfe).
+
+### How to Invite UHelper to Your Server
+
+![png](assets/0.png)
+
+If you can see a screen similar as above, that means you've successfully created your own server. 
+You are now ready to invite UHelper to help you through your journey as a student. Next, click this 
+[link](https://discord.com/oauth2/authorize?client_id=1253813199397191863&permissions=1126727225142272&integration_type=0&scope=bot). The link will take you to a webpage that not only informs you that you are about to 
```

Comment:
I think you can revise it to just say "The link will take you to a webpage that prompts you to select which server you want UHelper to join".

Reply:
Got it